src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -470,6 +470,8 @@
            ProcessFrameworkReferences not be able to restore for the given RID.  -->
       <BuildArgs Condition="'$(DotNetBuildSourceOnly)' != 'true' and '@(TransitiveRepositoryReference->AnyHaveMetadataValue('Identity', 'runtime'))' == 'true'">$(BuildArgs) /p:DotNetBuildTargetRidOnly=true</BuildArgs>
 
+      <BuildArgs>$(BuildArgs) /p:EnableDefaultRidSpecificArtifacts=$(EnableDefaultRidSpecificArtifacts)</BuildArgs>
```

Comment:
I would move this property out of the target next to the other properties that append to BuildArgs / CommonArgs. This target exists for dynamic properties that can't be statically defined.